### ISSUE TITLE
Unable to process command '::add-matcher::checkov-problem-matcher.json' successfully

### ISSUE THREAD

**Issue by AliAllomani** [bridgecrewio/checkov-action#81]
version: `v12.1478.0`

```
Error: Unable to process command '::add-matcher::checkov-problem-matcher.json' successfully.
Error: Could not find file '/home/runner/_work/XXXX/checkov-problem-matcher.json'.
```

```
Error parsing file bridgecrew-problem-matcher-warning.jsonֿ
Error parsing file bridgecrew-problem-matcher.jsonֿ
```

```
Run bridgecrewio/checkov-action@v12.1478.0
  with:
    directory: .
    output_format: sarif
    log_level: WARNING
    container_user: 0
/usr/bin/docker run --name bridgecrewcheckov201067_478f7b --label b1cbc5 --workdir /github/workspace --rm -e INPUT_DIRECTORY -e INPUT_CHECK -e INPUT_SKIP_CHECK -e INPUT_QUIET -e INPUT_API-KEY -e INPUT_SOFT_FAIL -e INPUT_FRAMEWORK -e INPUT_EXTERNAL_CHECKS_DIRS -e INPUT_EXTERNAL_CHECKS_REPOS -e INPUT_OUTPUT_FORMAT -e INPUT_DOWNLOAD_EXTERNAL_MODULES -e INPUT_LOG_LEVEL -e INPUT_CONFIG_FILE -e INPUT_BASELINE -e INPUT_SOFT_FAIL_ON -e INPUT_HARD_FAIL_ON -e INPUT_CONTAINER_USER -e API_KEY_VARIABLE -e HOME -e GITHUB_JOB -e GITHUB_REF -e GITHUB_SHA -e GITHUB_REPOSITORY -e GITHUB_REPOSITORY_OWNER -e GITHUB_RUN_ID -e GITHUB_RUN_NUMBER -e GITHUB_RETENTION_DAYS -e GITHUB_RUN_ATTEMPT -e GITHUB_ACTOR -e GITHUB_WORKFLOW -e GITHUB_HEAD_REF -e GITHUB_BASE_REF -e GITHUB_EVENT_NAME -e GITHUB_SERVER_URL -e GITHUB_API_URL -e GITHUB_GRAPHQL_URL -e GITHUB_REF_PROTECTED -e GITHUB_REF_TYPE -e GITHUB_WORKSPACE -e GITHUB_ACTION -e GITHUB_EVENT_PATH -e GITHUB_ACTION_REPOSITORY -e GITHUB_ACTION_REF -e GITHUB_PATH -e GITHUB_ENV -e GITHUB_STEP_SUMMARY -e RUNNER_OS -e RUNNER_ARCH -e RUNNER_NAME -e RUNNER_TOOL_CACHE -e RUNNER_TEMP -e RUNNER_WORKSPACE -e ACTIONS_RUNTIME_URL -e ACTIONS_RUNTIME_TOKEN -e GITHUB_ACTIONS=true -e CI=true -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/_work/_temp/_github_home":"/github/home" -v "/home/runner/_work/_temp/_github_workflow":"/github/workflow" -v "/home/runner/_work/_temp/_runner_file_commands":"/github/file_commands" -v "/home/runner/_work/XXXXX":"/github/workspace" bridgecrew/checkov:2.0.1067  "." "" "" "" "" "" "" "" "sarif" "" "WARNING" "" "" "" "" "--user 0"

Error: Unable to process command '::add-matcher::checkov-problem-matcher.json' successfully.
Error: Could not find file '/home/runner/_work/XXXX/checkov-problem-matcher.json'.

BC_FROM_BRANCH=actions
BC_TO_BRANCH=master
BC_PR_ID=106
BC_PR_URL=https://XXXXX/pull/106
.....
check_suite_focus=true#step:4:13)a71a8
....
BC_RUN_ID=8
....

running checkov on directory: .
checkov -d .      --output sarif


       _               _              
   ___| |__   ___  ___| | _______   __
  / __| '_ \ / _ \/ __| |/ / _ \ \ / /
 | (__| | | |  __/ (__|   < (_) \ V / 
  \___|_| |_|\___|\___|_|\_\___/ \_/  
                                      
By bridgecrew.io | version: 2.0.1067 
Update available 2.0.1067 -> 2.0.1068
Run pip3 install -U checkov to update 

terraform_plan scan results:

Passed checks: 0, Failed checks: 0, Skipped checks: 0, Parsing errors: 2

Error parsing file bridgecrew-problem-matcher-warning.jsonֿ
Error parsing file bridgecrew-problem-matcher.jsonֿ


Wrote output in SARIF format to the file 'results.sarif'

       _               _              
   ___| |__   ___  ___| | _______   __
  / __| '_ \ / _ \/ __| |/ / _ \ \ / /
 | (__| | | |  __/ (__|   < (_) \ V / 
  \___|_| |_|\___|\___|_|\_\___/ \_/  
                                      
By bridgecrew.io | version: 2.0.1067 
Update available 2.0.1067 -> 2.0.1068
Run pip3 install -U checkov to update 

terraform_plan scan results:

Passed checks: 0, Failed checks: 0, Skipped checks: 0, Parsing errors: 2

Error parsing file bridgecrew-problem-matcher-warning.jsonֿ
Error parsing file bridgecrew-problem-matcher.jsonֿ


Wrote output in SARIF format to the file 'results.sarif'
```

**Comment by nimrodkor** [comment]
Hey @AliAllomani !
Does this reproduce if you use the latest version?

Just trying to understand if this is a real bug, or just a long time without a version release

**Comment by AliAllomani** [comment]
Hey @nimrodkor, which version you're referring to ? the used action version `v12.1478.0` released 2 days ago and on the docker side the matcher code is the same in the master branch

https://github.com/bridgecrewio/checkov/blob/e8b4619ce9a72c7961cf6c9bc132212d8a7ff3df/github_action_resources/entrypoint.sh#L11-L14

**Comment by nimrodkor** [comment]
Weird, I'm not getting it on my test repo:
```
Run bridgecrewio/checkov-action@v1[2](https://github.com/nimrodkor/test_yor_action/runs/6077669789?check_suite_focus=true#step:4:2).1478.0
  with:
    directory: .
    output_format: sarif
    log_level: WARNING
    container_user: 0
/usr/bin/docker run --name bridgecrewcheckov201067_147a2e --label 294b[3](https://github.com/nimrodkor/test_yor_action/runs/6077669789?check_suite_focus=true#step:4:3)1 --workdir /github/workspace --rm -e INPUT_DIRECTORY -e INPUT_CHECK -e INPUT_SKIP_CHECK -e INPUT_QUIET -e INPUT_API-KEY -e INPUT_SOFT_FAIL -e INPUT_FRAMEWORK -e INPUT_EXTERNAL_CHECKS_DIRS -e INPUT_EXTERNAL_CHECKS_REPOS -e INPUT_OUTPUT_FORMAT -e INPUT_DOWNLOAD_EXTERNAL_MODULES -e INPUT_LOG_LEVEL -e INPUT_CONFIG_FILE -e INPUT_BASELINE -e INPUT_SOFT_FAIL_ON -e INPUT_HARD_FAIL_ON -e INPUT_CONTAINER_USER -e API_KEY_VARIABLE -e HOME -e GITHUB_JOB -e GITHUB_REF -e GITHUB_SHA -e GITHUB_REPOSITORY -e GITHUB_REPOSITORY_OWNER -e GITHUB_RUN_ID -e GITHUB_RUN_NUMBER -e GITHUB_RETENTION_DAYS -e GITHUB_RUN_ATTEMPT -e GITHUB_ACTOR -e GITHUB_WORKFLOW -e GITHUB_HEAD_REF -e GITHUB_BASE_REF -e GITHUB_EVENT_NAME -e GITHUB_SERVER_URL -e GITHUB_API_URL -e GITHUB_GRAPHQL_URL -e GITHUB_REF_NAME -e GITHUB_REF_PROTECTED -e GITHUB_REF_TYPE -e GITHUB_WORKSPACE -e GITHUB_ACTION -e GITHUB_EVENT_PATH -e GITHUB_ACTION_REPOSITORY -e GITHUB_ACTION_REF -e GITHUB_PATH -e GITHUB_ENV -e GITHUB_STEP_SUMMARY -e RUNNER_OS -e RUNNER_ARCH -e RUNNER_NAME -e RUNNER_TOOL_CACHE -e RUNNER_TEMP -e RUNNER_WORKSPACE -e ACTIONS_RUNTIME_URL -e ACTIONS_RUNTIME_TOKEN -e ACTIONS_CACHE_URL -e GITHUB_ACTIONS=true -e CI=true -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/work/_temp/_github_home":"/github/home" -v "/home/runner/work/_temp/_github_workflow":"/github/workflow" -v "/home/runner/work/_temp/_runner_file_commands":"/github/file_commands" -v "/home/runner/work/test_yor_action/test_yor_action":"/github/workspace" bridgecrew/checkov:2.0.1067  "." "" "" "" "" "" "" "" "sarif" "" "WARNING" "" "" "" "" "--user 0"
BC_FROM_BRANCH=test_pr
BC_TO_BRANCH=main
BC_PR_ID=2
BC_PR_URL=https://github.com/nimrodkor/test_yor_action/pull/2
BC_COMMIT_HASH=[4](https://github.com/nimrodkor/test_yor_action/runs/6077669789?check_suite_focus=true#step:4:4)[5](https://github.com/nimrodkor/test_yor_action/runs/6077669789?check_suite_focus=true#step:4:5)b42a223f9c78f0bab4ddd8ba0f55794f4bf1bf
BC_COMMIT_URL=https://github.com/nimrodkor/test_yor_action/commit/45b42a223f9c78f0bab4ddd8ba0f55794f4bf1bf
BC_AUTHOR_NAME=nimrodkor
BC_AUTHOR_URL=https://github.com/nimrodkor
BC_RUN_ID=4
BC_RUN_URL=https://github.com/nimrodkor/test_yor_action/actions/runs/21895537[6](https://github.com/nimrodkor/test_yor_action/runs/6077669789?check_suite_focus=true#step:4:6)2
BC_REPOSITORY_URL=https://github.com/nimrodkor/test_yor_action
running checkov on directory: .
checkov -d .      --output sarif       


       _               _              
   ___| |__   ___  ___| | _______   __
  / __| '_ \ / _ \/ __| |/ / _ \ \ / /
 | (__| | | |  __/ (__|   < (_) \ V / 
  \___|_| |_|\___|\___|_|\_\___/ \_/  
                                      
By bridgecrew.io | version: 2.0.106[7](https://github.com/nimrodkor/test_yor_action/runs/6077669789?check_suite_focus=true#step:4:7) 
Update available 2.0.[10](https://github.com/nimrodkor/test_yor_action/runs/6077669789?check_suite_focus=true#step:4:10)67 -> 2.0.1072
Run pip3 install -U checkov to update 

terraform scan results:
...
```

link - https://github.com/nimrodkor/test_yor_action/runs/6077669789?check_suite_focus=true

**Comment by AliAllomani** [comment]
@nimrodkor thank you for the test, this might be an issue with the self-hosted runner setup i tested on, will close this issue for now
